e2e/chains/bsc/Dockerfile.bootstrap

```diff
@@ -1,6 +1,6 @@
-FROM ghcr.io/foundry-rs/foundry:nightly-462b2ac6c038dc24b8f38b0c59b664d0740604c2
+FROM ghcr.io/foundry-rs/foundry:v0.3.0
```

Comment:
nightly-462b2ac6c038dc24b8f38b0c59b664d0740604c2 is no longer available

Reply:
Thank you! I was just encountering the same problem in another repository. When accessing with nightly-xxx, it becomes 'manifest unknown,' but I see that it can be retrieved when specified with a version